### PR TITLE
feat(files): Add new potions to brewing guide

### LINKED ISSUE #113: [Utility] Brewing Guide
### Discussed in https://github.com/BedrockTweaks/Files/discussions/104

<div type='discussions-op-text'>

<sup>Originally posted by **PAPISOP** June 26, 2024</sup>
i was planning to add new potions ingredients which were added in 1.21 to brewing guide but it looks like even vanilla tweaks didnt update it so idk what to do lol
like should i add it or wait for vanilla tweaks update then add it?</div>

Add new potions ingredient to brewing guide

### PR DESCRIPTION
**Pull request by PAPISOP**
Add new potions ingredient added in 1.21 to brewing guide
Readjusts the position of brewing guide guis in pocket gui (Because the gui was overlapping other guis in brewing stand and it was not looking good)

Resolves #113

By checking the following boxes with an X, you ensure that:

- [X] The pack was tested ingame in at least one device.
- [X] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [X] The pack code follows the style guide.
- [X] The commits follow the contribution guidelines.
- [X] The PR follows the contribution guidelines.

- [X] (Optional) Tested in Windows
- [ ] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS
[Screenshot 2024-06-27 194002](https://github.com/BedrockTweaks/Files/assets/85175982/af6b19b5-0a41-4fd9-a91c-82301338dbde)
Screenshot 2024-06-27 193940

### CODE REVIEW

**Comment by NSPC911** [comment]
couldnt the potion stuff be put below the inventory slots?

**Comment by PAPISOP** [comment]
> for pocket ui: couldnt the potion stuff be put below the inventory slots?

No because it will look like this removing the background of the gui

Screenshot 2024-06-28 124608

**Comment by SquatchHunter** [comment]
that doesnt look awful tbh

**Comment by NSPC911** [comment]
possible to make it horizontal instead of vertical?

File: resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui [hunk 1]
```diff
@@ -254,6 +254,14 @@
     "$item_texture": "textures/items/turtle_helmet",
     "$hover_text": "item.turtle_helmet.name"
   },
+  "breeze_rod@multi_item_panel": {
+    "$item_texture": "textures/items/breeze_rod",
+    "$hover_text": "item.breeze_rod.name"
+  },
+  "stone@multi_item_panel": {
+    "$item_texture": "textures/blocks/stone",
```
Comment (drav0011) [inline resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui:262]: change to a 3d item, can take a high res screenshot and edit the stone out

File: resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui [hunk 2]
```diff
@@ -278,6 +286,14 @@
     "$item_texture": "textures/items/spider_eye_fermented",
     "$hover_text": "item.fermented_spider_eye.name"
   },
+  "cobweb@multi_item_panel": {
+    "$item_texture": "textures/blocks/web",
+    "$hover_text": "tile.web.name"
+  },
+  "slime_block@multi_item_panel": {
+    "$item_texture": "textures/blocks/slime",
```
Comment (drav0011) [inline resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui:294]: same as stone

**Review by drav0011 — CHANGES_REQUESTED** [review]

**Comment by NSPC911** [comment]
> in 21:9 mobiles will looks a lot different; iphones, different... 

~~screw iphones~~

File: resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui (hunk 1 above)
Comment (PAPISOP) [reply resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui]: Done, and the textures are 114x126 and i used blockbench to render it
Screenshot 2024-06-28 223932

File: resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui (hunk 2 above)
Comment (PAPISOP) [reply resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui]: See the above reply

**Comment by PAPISOP** [comment]
> possible to make it horizontal instead of vertical?

I don't think so that it can become horizontal (Need to completely make an other gui for pocket gui) and it won't be same as vanilla tweaks

**Review by drav0011 — APPROVED** [review]
lgtm
- [x] (Optional) Tested in Windows

**Pull request merged**